### ISSUE TITLE
current issues with doc

### ISSUE THREAD

**Issue by zelig** [ethersphere/swarm-guide#58]
* [x] http://swarm-docs.s3-eu-west-1.amazonaws.com/up-and-download.html#adding-entries-to-a-manifest this needs proper examples
* [x] swarm-docs.s3-eu-west-1.amazonaws.com/up-and-download.html#downloading-swarm-content-from-your-local-node  swarm down should be part of the CLI https://github.com/ethersphere/go-ethereum/issues/439
* [x] http://swarm-docs.s3-eu-west-1.amazonaws.com/up-and-download.html#tar-stream-upload this should have explanation/text
* [x] http://swarm-docs.s3-eu-west-1.amazonaws.com/userguide.html#fuse FUSE section is incorrectly under resource updated (install FUSE appears in the sidebar )
* [x] FUSE need explanation, text
* [ ] PSS - there is no example, user doesn't even know how i can interact with this component
* [x] cross referencing 
  * between usage and API ref
  * CLI and http for upload/download
  * etc
* [x] Introduced in POC 0.3, symmetric encryption using SHA3 Keccak256 is ...... putting the concepts of plausible deniability and censorship resistence to work. this makes no sense like this
* [x] the entire confifguration section contains outdated stuff. how come.
*  [x] why are the private and singleton modes in the config section? Not sure i  like this reorganisation . 
* [x] URL schemes are duplicated in the doc, but first it says: Swarm offers 8 distinct URL schemes. No it  is 6, deprecated stuff should go
* [x] all thigs with SWAP should go until we got it done
* [x] why is encryption a separate section while MRU and FUSE are hidden under working with content
* [x] what is this duplicate http://swarm-docs.s3-eu-west-1.amazonaws.com/apireference.html#bzz-schemes doing in the API ref section

**Comment by homotopycolimit** [comment]
Going through this top to bottom.
Sections that are "almost ready" are 
Introduction
Installation and Updates [still needs binary install]
Getting Started
Uploading and Downloading [needs `swarm down`]
Using ENS names
Encryption
BZZ URL Schemes

the rest is still a jumble of things from earlier versions. parts are fine and others are not. 
For example:
"why are the private and singleton modes in the config section? Not sure i like this reorganisation ."
In some sense, they were always part of this section, it's just that other stuff was moved out so now it is noticeably out of place. So yeah, let's move them.. but where to?


Fuse needs actual documentation - how do you use FUSE? we need `swarm fs mount` `swarm fs unmount` and `swarm fs list` commands implemented and documented - https://github.com/ethersphere/go-ethereum/issues/438

"why is encryption a separate section while MRU and FUSE are hidden under working with content"

because I was in the middle of a cleanup. this is not final. In particular "working with content" is a hodgepodge of older sections.

**Comment by justelad** [comment]
@homotopycolimit regarding fuse that's exactly what i'm blocked on.
there's a lot of IPC commands involved. when i talked with @zelig and @nolash about this, they said that FUSE should be used through IPC and that this is the intention, there was no mentioning about integrating it into the CLI but rather leaving it on IPC.
The same goes for PSS, I have no idea why IPC is the preferred way of doing this, but talking to @nolash he also clarified that there's no plans on having this either on CLI or HTTP API. hence from my side there's nothing more to add and write over these section but a walkthrough of how to use it though the IPC commands.

**Comment by homotopycolimit** [comment]
"they said that FUSE should be used through IPC"
That makes no sense to me. What's the reasoning behind that? 

"to mount this manifest as a directory do `geth attach /path/to/bzzd.ipc` and then, in the console, type `swarmfs.mount(hash,path)`"

vs.

"to mount this manifest as a directory type `swarm fs mount MANIFEST` 

PSS I can understand - because it is low level and is supposed to be used by software and not by people, but FUSE mounting a manifest is very much something I'd want to be able to do as a user. Or am I missing something here?

**Comment by emazurek** [comment]
> The most up-to-date swarm book in various formats is available on the old web http://ethersphere.org/swarm/docs as well as on swarm bzz://swarm/guide

FYI, that URL is broken

**Comment by zelig** [comment]
@homotopycolimit @justelad sure fuse should have a cli. please write it :)
it needs explanation of usage

ipc should be json rpc as i said, the sentence about bzz should go,

url schemes still says 8 not 6

MRU double embedding

alternative networks is under getting started? thats why the section was running a node. why change it.

the rest until api ref should come under usage right? why separate toplevel  chapters ?

encryption bit is poor @gbalint @nagydani 

@nolash pss needs subscribe section, examples, api ref

**Comment by justelad** [comment]
can i close this? not sure this is any longer relevant

**Comment by petyaracz** [comment]
the PSS documentation is still patchy, it looks like the rest of the boxes been ticked? i could open a separate issue for that and then you can close this?
